Src/Particle/AMReX_Particle.H

```diff
@@ -188,6 +188,12 @@ struct ConstParticleCPUWrapper
     operator int () const noexcept { return static_cast<int>(m_idata & 0x00FFFFFF); }
 };
 
+std::uint64_t SetParticleIDandCPU (Long id, int cpu) {
```

Comment:
```suggestion
/** Set the idcpu value at once, based on a particle id and cpuid
 *
  * This can be used in initialization & assignments to avoid writing twice into the same memory bank.
 */
std::uint64_t SetParticleIDandCPU (Long id, int cpu) {
```